swift_browser_ui_frontend/src/common/lang.js

```diff
@@ -46,10 +46,11 @@ let default_translations = {
         inUse: "Bucket name already in use.",
         invalidName: "Bucket name is invalid.",
         createFail: "Bucket creation failed.",
-        idb: "Browser not supported.",
+        idb: "Firefox in private mode is not supported.",
```

Comment:
It looks like you are not using this string anywhere

Reply:
It's used in IndexPage as a card title

Reply:
Why is it different in `IndexPage`?